Test payment missing calls.

wwjfrodriguez opened the pull request:
Testing `test_create_chargeback_and_chargeback_reversal`  and `test_payment_refund_by_external_key`

wwjfrodriguez committed:
Test payment missing calls.

wwjfrodriguez committed:
Change method name

pierre reviewed (approved):
Will merge once the requested changes in the Ruby client have been implemented.

wwjfrodriguez committed:
Fix chargerbacks test

wwjfrodriguez committed:
Fix tests: 'test_create_chargeback_and_chargeback_reversal', and 'test_payment_refund_by_external_key'

wwjfrodriguez committed:
Remove blank spaces.
